test/services-test.js

```diff
@@ -0,0 +1,558 @@
+'use strict';
+
+const testUrl = 'http://localhost:3000';
+
```

Comment:
Thanks Duncan, I originally tried them separately but they kept having conflicts... so I lumped them together.  I will look at the code later and see what caused the conflict and how to separate the testing.  Thanks for all you have done!  It was a pleasure to have you as an instructor!